src/plugins/intel_gpu/src/kernel_selector/kernels/gather/gather_nd_kernel_ref.cpp

```diff
@@ -177,13 +201,35 @@ KernelsData GatherNDKernelRef::GetKernelsData(const Params& params, const option
     KernelData kd = KernelData::Default<gather_nd_params>(params);
     gather_nd_params& newParams = *static_cast<gather_nd_params*>(kd.params.get());
 
-    auto dispatchData = SetDefault(newParams, options);
```

Comment:
why this changed is needed?

Reply:
This is not critical, I just followed gather case(https://github.com/openvinotoolkit/openvino/pull/13987).

Reply:
Not only gather but also several kernels merged recently have same change. I think it is better to sync with them.